app/src/main/java/de/westnordost/streetcomplete/quests/shop_type/ShopGoneDialog.kt

```diff
@@ -68,7 +68,7 @@ class ShopGoneDialog(
         getButton(DialogInterface.BUTTON_POSITIVE).setOnClickListener {
             when (selectedRadioButtonId) {
                 R.id.vacantRadioButton -> {
-                    onSelectedFeature(mapOf("shop" to "vacant"))
+                    onSelectedFeature(mapOf("disused:shop" to "yes"))
```

Comment:
Reading the code we had:
`shop=supermarket` => `shop=vacant`
and now we have:
`shop=supermarket` => unchanged
**added** `disused:shop=yes`

I think we should do
`shop=supermarket` => `disused:shop=yes`

Reply:
definitely, not sure how I managed to miss it

Reply:
Or maybe it actually works? Looking again at code it works the same way as other feature replacement (`onSelectedFeature(feature.addTags)`)

Yeah, it works.

`D/QuestController: Replaced NODE #6325607440 in frame of quest AddOpeningHours with DELETE "shop"="gift", DELETE "noname"="yes", ADD "disused:shop"="yes"`